app/controllers/organizations_controller.rb

```diff
@@ -1,17 +1,18 @@
 class OrganizationsController < ApplicationController
   before_action :authenticate_user!
+  load_and_authorize_resource
   layout 'org_admin'
 
   def show
     @org = Organization.friendly.find(params[:id])
     session[:current_campaign] ||= Campaign.current_campaign_id(params[:id])
-    if session[:current_campaign]
+    if session[:current_campaign] == -1
+      @organization_campaign = nil
+    else
```

Comment:
This could be more simply:

```
@organization_campaign = OrganizationCampaign.find_by(
  organization_id: @org.id,
  campaign_id: session[:current_campaign] # see previous comment about the session
)
```

`find_by` will [return nil](http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by) if no record is found